src/hooks/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import { AuthProvider } from './useAuth';
+
+const AppProvider: React.FC = ({ children }: any) => (
```

Comment:
```suggestion
const AppProvider = ({ children }: JSX.Element) => (
```
- Por problemas de intellisense, não está mais sendo recomendado o uso da tipagem `React.FC`;
- Acho que seria uma boa tiparmos a prop `children` mas não tenho certeza de qual tipagem seria a correta. Pode testar a sugerida acima? 🙂

Reply:
Isso! O ideal seria tipar o children como o `JSX.Element`

Reply:
Show... passou batido aqui. Vi que estão usando o JSX.Element mesmo.